cellarium/cas/data_preparation/callbacks.py

```diff
@@ -30,19 +31,28 @@ def calculate_total_mrna_umis(adata: "anndata.AnnData") -> None:
         >>> 'total_mrna_umis' in adata.obs.columns
         True.
     """
-    adata.obs[TOTAL_MRNA_UMIS_COLUMN_NAME] = np.array(adata.X.sum(axis=1)).flatten()
+    if count_matrix_name not in {"X", "raw.X"}:
+        raise ValueError("`count_matrix_name` should have a value of either 'X' or 'raw.X'.")
 
+    if count_matrix_name == "raw.X":
+        count_matrix = adata.raw.X
+    else:
+        count_matrix = adata.X
 
-_PRE_SANITIZE_CALLBACKS: t.List[t.Callable[["anndata.AnnData"], None]] = [calculate_total_mrna_umis]
+    adata.obs[TOTAL_MRNA_UMIS_COLUMN_NAME] = np.array(count_matrix.sum(axis=1)).flatten()
 
 
-def pre_sanitize_callback(adata: "anndata.AnnData") -> None:
+_PRE_SANITIZE_CALLBACKS: t.List[t.Callable[["anndata.AnnData", str], None]] = [calculate_total_mrna_umis]
```

Comment:
does `"anndata.AnnData"` need to be in quotes?

Reply:
Actually, not, that's a bad habit I have... Thanks!